src/components/Inventory.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Checkbox from './Checkbox.js';
+import InputField from './InputField.js';
+
+const labelList = {
+  OnOff: 'On/Off',
```

Comment:
I really like this, because the first thing it makes me think of is i18n and l10n.  This game is built in Europe, so lots of languages :-) Those are very far past mvp, but I appreciate the fact that it should be easy to do with the labelList being const's.